Source/View Controller/ScheduleTableViewController.swift

```diff
@@ -85,6 +96,22 @@ internal class ScheduleTableViewController: NavigationBarViewController, Navigat
         return cell
     }
     
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let destinationID = "MapViewController"
+        
+        guard let mapViewController = self.storyboard?.instantiateViewController(withIdentifier: destinationID) as? MapViewController
+            else {
+                print("ERROR: View controller not found")
+                return
+        }
+        
+        // TODO: Pass map image url to mapViewController
+        //let mapImageURL = viewModel.viewContent[indexPath.section][indexPath.row].mapImage
+        //mapViewController.setImage(mapImageURL)
```

Comment:
Good looking out. You can take out the "TODO" part of this comment because it will trigger a warning

Reply:
I separated the TODO into TO DO. I was also confused as to why swift lint does that.